plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -24,36 +24,46 @@
 - "Naoya Nakazawa (@n0ts)"
 options:
     api_key:
+        type: str
         description: ["Your DataDog API key."]
         required: true
     app_key:
+        type: str
         description: ["Your DataDog app key."]
         required: true
     title:
+        type: str
         description: ["The event title."]
         required: true
     text:
+        type: str
         description: ["The body of the event."]
         required: true
     date_happened:
+        type: int
         description:
         - POSIX timestamp of the event.
         - Default value is now.
-        default: now
     priority:
+        type: str
         description: ["The priority of the event."]
         default: normal
         choices: [normal, low]
     host:
-        description: ["Host name to associate with the event."]
-        default: "{{ ansible_hostname }}"
+        type: str
+        description:
+        - Host name to associate with the event.
+        - If not specified, it defaults to the value of C(ansible_hostname).
```

Comment:
This is just wrong on so many levels. The module has absolutely no idea what `ansible_hostname` is. It can try to identify the hostname of the system it runs on, or one of its IP addresses, but it cannot know if these are in any way related to the value of `ansible_hostname`.

How about writing instead:
```suggestion
        - If not specified, it defaults to the remote system's hostname.
```
(I know this is not the scope of this PR, but... :) )